lnbits/wallets/fake.py

```diff
@@ -42,44 +51,55 @@ async def create_invoice(
         memo: Optional[str] = None,
         description_hash: Optional[bytes] = None,
         unhashed_description: Optional[bytes] = None,
-        **kwargs,
+        expiry: Optional[int] = None,
+        payment_secret: Optional[bytes] = None,
+        **_,
     ) -> InvoiceResponse:
-        data: Dict = {
-            "out": False,
```

Comment:
- `out` is no longer used, is that ok?

Reply:
yes, it was unused, at least my tests worked fine.